wled00/cfg.cpp

```diff
@@ -643,6 +643,8 @@ bool deserializeConfig(JsonObject doc, bool fromFS) {
     CJSON(wifiLock, ota[F("lock-wifi")]);
     CJSON(aOtaEnabled, ota[F("aota")]);
     getStringFromJson(otaPass, pwd, 33); //normally not present due to security
+    interfaceUpdateCallMode = CALL_MODE_WS_SEND;
```

Comment:
I do not think it is necessary to add this here as cfg is only read at boot time.